python/cog/server/http.py

```diff
@@ -241,6 +229,25 @@ def start_shutdown() -> Any:
         shutdown_event.set()
         return JSONResponse({}, status_code=200)
 
+    def _check_setup_result() -> Any:
+        if app.state.setup_result is None:
+            return
+
+        if not app.state.setup_result.ready():
+            return
+
+        result = app.state.setup_result.get()
```

Comment:
`AsyncResult` will allow you to consume the result repeatedly, right? It seems like it's just a wrapper object so I'm sure it's fine

Reply:
Yup, this is fine to call repeatedly.